src/acvp.h

```diff
@@ -251,6 +254,54 @@ typedef enum acvp_drbg_pre_req {
     DRBG_TDES
 } ACVP_DRBG_PRE_REQ;
 
+
+#define ACVP_RSA_PRIME_SHA_1         "SHA-1"
+#define ACVP_RSA_PRIME_SHA_224       "SHA-224"
+#define ACVP_RSA_PRIME_SHA_256       "SHA-256"
+#define ACVP_RSA_PRIME_SHA_384       "SHA-384"
+#define ACVP_RSA_PRIME_SHA_512       "SHA-512"
+#define ACVP_RSA_PRIME_SHA_512_224   "SHA-512/224"
+#define ACVP_RSA_PRIME_SHA_512_256   "SHA-512/256"
+
+typedef enum acvp_rsa_param {
+    ACVP_PUB_EXP = 0,
+    ACVP_FIXED_PUB_EXP_VAL,
+    ACVP_RAND_PQ,
+    ACVP_CAPS_PROV_PRIME,
+    ACVP_CAPS_PROB_PRIME,
+    ACVP_CAPS_PROV_PROB_PRIME,
+    ACVP_RSA_INFO_GEN_BY_SERVER
+} ACVP_RSA_PARM;
+
+#define MOD_PRIME_2048     2048
+#define MOD_PRIME_3072     3072
+#define MOD_PRIME_4096     4096
+
+#define PROB_PRIME_TEST_2       2
+#define PROB_PRIME_TEST_3       3
+
+#define PRIME_TEST_TBLC2_NAME "tblC2"
+#define PRIME_TEST_TBLC3_NAME "tblC3"
+
+#define RSA_RAND_PQ_B32        1
```

Comment:
I was referring to the spec.  Section 2.4.1.3 we use {"1","2","3","4","5"} instead of being more explicit. 
Maybe that changed ?
Just ignore this for now, it has nothing to do with the code you wrote and more to do with the spec.